<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="gkillare-win87" failures="0" tests="38" name="testcases.processorfailourchase.TC_ProcessorFailour13" time="2.156" errors="1" timestamp="2024-03-22T15:39:38 IST" skipped="37">
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="setHeadlines" time="2.156">
    <error message="Error on line 1: XML document structures must start and end within the same entity." type="org.jdom2.input.JDOMParseException">
      <![CDATA[org.jdom2.input.JDOMParseException: Error on line 1: XML document structures must start and end within the same entity.
at org.jdom2.input.sax.SAXBuilderEngine.build(SAXBuilderEngine.java:232)
at org.jdom2.input.sax.SAXBuilderEngine.build(SAXBuilderEngine.java:303)
at org.jdom2.input.SAXBuilder.build(SAXBuilder.java:1196)
at responsevalidator.Response_Parameters.<init>(Response_Parameters.java:17)
at testcases.processorfailourchase.TC_ProcessorFailour13.setHeadlines(TC_ProcessorFailour13.java:68)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 194; XML document structures must start and end within the same entity.
at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.fatalError(ErrorHandlerWrapper.java:178)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:400)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:327)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLScanner.reportFatalError(XMLScanner.java:1465)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.endEntity(XMLDocumentFragmentScannerImpl.java:943)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.endEntity(XMLDocumentScannerImpl.java:562)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLEntityManager.endEntity(XMLEntityManager.java:1503)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLEntityScanner.load(XMLEntityScanner.java:1720)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLEntityScanner.skipChar(XMLEntityScanner.java:1375)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2762)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:605)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:112)
at java.xml/com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:542)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:889)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:825)
at java.xml/com.sun.org.apache.xerces.internal.parsers.XMLParser.parse(XMLParser.java:141)
at java.xml/com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse(AbstractSAXParser.java:1224)
at java.xml/com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse(SAXParserImpl.java:637)
at org.jdom2.input.sax.SAXBuilderEngine.build(SAXBuilderEngine.java:217)
... 30 more
]]>
    </error>
  </testcase> <!-- setHeadlines -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="DebitSale" time="0.000">
    <skipped/>
  </testcase> <!-- DebitSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="DebitSale" time="0.000">
    <skipped/>
  </testcase> <!-- DebitSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_FSA" time="0.000">
    <skipped/>
  </testcase> <!-- P_FSA -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="Ewic_Transactions" time="0.000">
    <skipped/>
  </testcase> <!-- Ewic_Transactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_EBT" time="0.000">
    <skipped/>
  </testcase> <!-- P_EBT -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_EBT" time="0.000">
    <skipped/>
  </testcase> <!-- P_EBT -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="F_creditSale" time="0.000">
    <skipped/>
  </testcase> <!-- F_creditSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="F_creditSale" time="0.000">
    <skipped/>
  </testcase> <!-- F_creditSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="F_creditSale" time="0.000">
    <skipped/>
  </testcase> <!-- F_creditSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_HardCode__FSA" time="0.000">
    <skipped/>
  </testcase> <!-- P_HardCode__FSA -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_HardCode__EBT" time="0.000">
    <skipped/>
  </testcase> <!-- P_HardCode__EBT -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="SECOUNDARY_creditAndDebitSale" time="0.000">
    <skipped/>
  </testcase> <!-- SECOUNDARY_creditAndDebitSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="SECOUNDARY_creditAndDebitSale" time="0.000">
    <skipped/>
  </testcase> <!-- SECOUNDARY_creditAndDebitSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="SECOUNDARY_creditAndDebitSale" time="0.000">
    <skipped/>
  </testcase> <!-- SECOUNDARY_creditAndDebitSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_SECOUNDARY__FSA" time="0.000">
    <skipped/>
  </testcase> <!-- P_SECOUNDARY__FSA -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_SECOUNDARY__EBT" time="0.000">
    <skipped/>
  </testcase> <!-- P_SECOUNDARY__EBT -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_SECOUNDARY__EBT" time="0.000">
    <skipped/>
  </testcase> <!-- P_SECOUNDARY__EBT -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="VoidAllTransactions" time="0.000">
    <skipped/>
  </testcase> <!-- VoidAllTransactions -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="ResetCount" time="0.000">
    <skipped/>
  </testcase> <!-- ResetCount -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="afterResetDebitSale" time="0.000">
    <skipped/>
  </testcase> <!-- afterResetDebitSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="afterResetDebitSale" time="0.000">
    <skipped/>
  </testcase> <!-- afterResetDebitSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_AfterReset_EBT" time="0.000">
    <skipped/>
  </testcase> <!-- P_AfterReset_EBT -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_AfterReset_EBT" time="0.000">
    <skipped/>
  </testcase> <!-- P_AfterReset_EBT -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="P_AfterReset_FSA" time="0.000">
    <skipped/>
  </testcase> <!-- P_AfterReset_FSA -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="F_AfterReset_creditSale" time="0.000">
    <skipped/>
  </testcase> <!-- F_AfterReset_creditSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="F_AfterReset_creditSale" time="0.000">
    <skipped/>
  </testcase> <!-- F_AfterReset_creditSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="F_AfterReset_creditSale" time="0.000">
    <skipped/>
  </testcase> <!-- F_AfterReset_creditSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="F_AfterReset_creditSale" time="0.000">
    <skipped/>
  </testcase> <!-- F_AfterReset_creditSale -->
  <system-out/>
  <testcase classname="testcases.processorfailourchase.TC_ProcessorFailour13" name="DebitSaleLastOne" time="0.000">
    <skipped/>
  </testcase> <!-- DebitSaleLastOne -->
  <system-out/>
</testsuite> <!-- testcases.processorfailourchase.TC_ProcessorFailour13 -->
